src/gmail.js

```diff
@@ -3984,6 +4036,19 @@ var Gmail = function(localJQuery) {
                 show_cc: "span.aB.gQ.pE",
                 show_bcc: "span.aB.gQ.pB"
             };
+
+            // Post "peoplekit"
+            if (this.$el.find("textarea[name=to]").length === 0 && ["to", "cc", "bcc"].includes(lookup)) {
```

Comment:
Would be possible to extract the "peoplekit" check into another method, and *if* we are doing people-kit we patch up the config, no matter what the current lookup is?